api/lambdas/src/sqs-to-converter.ts

```diff
@@ -319,30 +345,34 @@ export const handler = Sentry.AWSLambda.wrapHandler(async event => {
             params,
             headers: { "Content-Type": "text/plain" },
           });
-          conversionResult = res.data;
+          conversionResult = res.data.fhirResource;
```

Comment:
"Unwrapped" the `fhirResource` so we work directly with `FHIRBundle`, we unwrapped in all functions anyways, had to artificially add it for sidechain, and it would require a wrapping type.